gitlab-pipeline/stage/test.yml

```diff
@@ -155,11 +155,15 @@ test:backend-integration:open_source:
 
 test:backend-integration:enterprise:
   extends: test:backend-integration:open_source
+  tags:
+    - mender-qa-worker-backend-integration-tests
```

Comment:
Why can't we use `hetzner-amd-beefy` for the enterprise tests?

Reply:
Because it doesn't work. I didn't invest too much time on checking it, with this PR I wanted just to move minutes out of GCP easily
https://gitlab.com/Northern.tech/Mender/mender-qa/-/jobs/6706320738#L939